npc/re/merchants/malangdo_costume.txt

```diff
@@ -100,6 +102,8 @@ mal_in01,20,124,5	script	Designer Heidam#eventhat	4_CAT_SAILOR5,{
 		20242,		// C_Snownow_Hat
 		20273,		// C_Soft_Sheep_Hat
 		20762;		// C_GreatDevilWing
+		19969;		// C_Blazing_Soul
+		19974;		// C_Camouflage_RabbitHood
```

Comment:
It looks like the following items are also missing from Box 6:
- Costume: Star Reading Hat - 20113
- Costume: Overprotector - 20283
- Costume: Iron Cain - 19528
- Costume: Red Under Rimmed Glasses - 20286
- Costume: Snowman Hat - 20231
- Costume: Puppy Hat - 20227
- Costume: Fried Egg Hat - 20160
- Costume: Silver Question - 19980
- Costume: Golden Question - 19979
- Costume: Silver Exclamation - 19978
- Costume: Golden Exclamation - 19977
- Costume: Blazing Soul - 19969
- Costume: Camouflage Bunny Hood - 19974
- Costume: Antler - 19939

![image](https://user-images.githubusercontent.com/7020503/103669836-f3f87080-4fcc-11eb-8f0d-d5a7d9db403a.png)

Reply:
For some reason my changes didn't come through. Re-adjusted and added the others in.

How 'official' is this repository of information? https://forums.warpportal.com/index.php?/topic/242220-updated-december-malangdo-costume-enchants/#entry2897199

If so, there's a massive list of costumes and there's no mention of the 4 I have highlighted for adding (Blazing Soul, Rabbit Hood, Driver Band, Magic Stone)

Reply:
Ah good question. It looks to be a reliable source for iRO's take on Costume Enchanting.
I suppose the path of least resistance is to go with what we have now. Then, if there is a desire for additional items we can address at that point in time.

Reply:
Agreed, I don't like the reliability of the information.

Also, we have no means of obtaining these costumes in order to utilise the system. Closing.